plugins/action/common/prepare_plugins/prep_108_route_control.py

```diff
@@ -0,0 +1,318 @@
+                                new_group = {
+                                    "name": unique_name,
+                                    "policies": [
+                                        {"name": unique_name},
+                                    ],
+                                    "priority": 500
+                                }
+                                model_data["vxlan"]["policy"]["groups"].append(new_group)
+
+            model_data = hostname_to_ip_mapping(model_data)
+        self.kwargs['results']['model_extended'] = model_data
+        return self.kwargs['results']
+
+    def route_maps(self, model_data):
+        """function to rewrite parameters in route_maps"""
+        for route_map in model_data["vxlan"]["overlay_extensions"]["route_control"]["route_maps"]:
+            if "entries" in route_map:
+                for entry in route_map["entries"]:
+                    if "match" in entry:
+                        for option_match in entry["match"]:
+                            # Rewrite match interface based on CLI
+                            # Order is not important, only the case
+                            # Example: CLI match interface Ethernet1/10 loopback100 Null0 port-channel100
+                            if "interface" in option_match:
+                                option_match["interface"] = self.rewrite_match_interface(
+                                    option_match["interface"])
+                    if "set" in entry:
+                        for option_set in entry["set"]:
+                            # Rewrite route-map IP Precedence number to string
+                            # Example: set ip precedence 6 -> set ip precedence internet
+                            precedence_translation = {
+                                0: 'routine',
+                                1: 'priority',
+                                2: 'immediate',
+                                3: 'flash',
+                                4: 'flash-override',
+                                5: 'critical',
+                                6: 'internet',
+                                7: 'network',
+                            }
+                            if ("ipv4" in option_set) and ("precedence" in option_set["ipv4"]):
+                                if isinstance(option_set["ipv4"]["precedence"], int):
+                                    option_set["ipv4"]["precedence"] = precedence_translation[
+                                        option_set["ipv4"]["precedence"]]
+
+                            if ("ipv6" in option_set) and ("precedence" in option_set["ipv6"]):
+                                if isinstance(option_set["ipv6"]["precedence"], int):
+                                    option_set["ipv6"]["precedence"] = precedence_translation[
+                                        option_set["ipv6"]["precedence"]]
+
+    def ipv4_access_list(self, model_data):
```

Comment:
It looks like `ipv4_access_list` and `ipv6_access_list` functions are almost identical.  Could you just use one function instead of two and pass in a key (either `ipv4_access_lists`, or `ipv6_access_lists`?

Reply:
Done. Refactor with one function: update_ip_access_lists()